k8s.io/configmap-nginx.yaml

```diff
@@ -90,6 +90,18 @@ data:
         rewrite ^/(.*)?$    https://github.com/kubernetes/kubernetes/tree/master/$1 redirect;
       }
       server {
+        server_name dl.k8s.io dl.kubernetes.io;
+        listen 80;
+        listen 443 ssl;
+
+        location / {
+          # Don't require /release/ if you want to get at the Kubernetes release artifacts, the common case.
+          rewrite ^/(v[0-9]+\.[0-9]+.*)$   https://storage.googleapis.com/kubernetes-release/release/$1 redirect;
```

Comment:
yeah, i was treating GCS as a file server again, and it just isn't.  I need GCSweb.